### PR TITLE
New package: open-vm-tools-10.1.15

### PR DESCRIPTION
**Pull request by memchk**
This closes issue #5502, and obsoletes PR #5347. I want to acknowlage @ottbot, this PR is heavily based on his, with an update and tweaks. 

I examined the possiblillity of using nmap's libdnet and for now the observed improvments do not outweigh the difficuitly of packaging it. This could be considered for down the road.

### CODE REVIEW

File: srcpkgs/libdnet/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'libdnet'
+pkgname=libdnet
+version=1.12
+revision=1
+makedepends="automake libtool autoconf pkg-config"
```
Comment (maxice8) [inline srcpkgs/libdnet/template]: automake, libtoo, autoconf and pkg-config should be in hostmakedepends to allow cross-compilation.

**Comment by memchk** [comment]
open-vm-tools I believe was never designed to work with ARM. For musl, additional work will need to be done as musl does not have the Sun RPC implementation. Would it be OK to set only_archs="i686 x86_64"?

**Comment by maxice8** [comment]
Alpine has the package and 15 patches bundled with it

**Comment by memchk** [comment]
Ok, I will see what I can do.

**Comment by maxice8** [comment]
The patches are over 9k lines and need to be ported to a new version, i'd honestly just mark only x86_64 i686 and work on fixing musl later.

**Comment by Vaelatern** [comment]
Please do not use `only_for_archs`. Better to mark where it is broken.

**Comment by memchk** [comment]
What is the appropriate flag for excluding certain Arch's? I did not see it
in the manual.

**Comment by Johnnynator** [comment]
@memchk mostly a case statement for marking broken builds get used e.g.
```
case "$XBPS_TARGET_MACHINE" in
    *-musl) broken="Musl is broken because stuff";;
    arm*) broken=yes
esac
```

**Comment by jsumners** [comment]
Awesome! This looks great. Thank you very much @memchk

**Comment by memchk** [comment]
Ok, I removed only_archs and added the suggested case statement. It looks like everything is building just fine at this point.

**Comment by memchk** [comment]
@maxice8 Is there anything further you need to see / I need to do before this merges?

**Comment by maxice8** [comment]
LGTM, now just need to wait for a Member to merge this.

**Comment by memchk** [comment]
Gotcha. One more thing, I have been updating my local copy of void-packages with `git pull --rebase upsteam master`, that is the proper way to do it right?

**Pull request closed without merging**